.github/workflows/ci.yaml

```diff
@@ -41,10 +54,13 @@ jobs:
           echo "and it will start over with a clean cache."
           echo "The old one will disappear after 7 days."
 
+      - name: Java Linter
+        run: bazel --batch run //:google_java_format_verification
+
       - name: Unit Tests
         run: bazel --batch test //:units --noshow_progress --test_output=errors
 
-      - name: Integration Tests
+      - name: Golden Tests
```

Comment:
Can we still reference `Integration` somewhere in the name? Some unit tests are golden tests too. Maybe something like `Golden Integration Tests`?

Reply:
If you want to keep the 'Integration' word, we can. I'm trying to move away from its use due to it not having a consistent definition across projects. 
* Client libraries use 'integration test' to refer to large tests with broad scope. 
* These golden tests are small tests with medium scope. 
* The showcase tests are medium tests with medium scope.

Yet all are "integration". At some point it loses its meaning.